ECommerceAPI/Infrastructure/Configurations/EmailConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ECommerceAPI.Infrastructure.Configurations
+{
+    public class EmailConfiguration
+    {
+        public string From { get; set; } = null!;
+        public string SmtpServer { get; set; } = null!;
+        public int Port { get; set; }
+        public string UserName { get; set; } = null!;
+        public string Password { get; set; } = null!;
```

Comment:
string by default is null

Reply:
You're right, but I used the null-forgiving operator (null!)